test/cmdlineTests/exp_base_literal/output

```diff
@@ -199,78 +199,157 @@ object "C_81" {
             }
 
             function fun_f_80(var_a_4, var_b_6, var_c_8, var_d_10) -> var__13, var__15, var__17, var__19 {
+                /// @origin exp_base_literal/input.sol:160,164
                 let zero_t_uint256_1 := zero_value_for_split_t_uint256()
                 var__13 := zero_t_uint256_1
+                /// @origin exp_base_literal/input.sol:166,169
                 let zero_t_int256_2 := zero_value_for_split_t_int256()
                 var__15 := zero_t_int256_2
+                /// @origin exp_base_literal/input.sol:171,175
                 let zero_t_uint256_3 := zero_value_for_split_t_uint256()
                 var__17 := zero_t_uint256_3
+                /// @origin exp_base_literal/input.sol:177,181
                 let zero_t_uint256_4 := zero_value_for_split_t_uint256()
                 var__19 := zero_t_uint256_4
 
+                /// @origin exp_base_literal/input.sol:196,200
+                /// @origin exp_base_literal/input.sol:196,197
                 let expr_23 := 0x02
+                /// @origin exp_base_literal/input.sol:199,200
+                /// @origin exp_base_literal/input.sol:199,200
                 let _5 := var_a_4
                 let expr_24 := _5
+                /// @origin exp_base_literal/input.sol:196,200
                 let _6 := convert_t_rational_2_by_1_to_t_uint256(expr_23)
                 let expr_25 := checked_exp_t_rational_2_by_1_t_uint256(expr_24)
+                /// @origin exp_base_literal/input.sol:187,200
                 let var_w_22 := expr_25
+                /// @origin exp_base_literal/input.sol:212,219
+                /// @origin exp_base_literal/input.sol:212,216
+                /// @origin exp_base_literal/input.sol:214,215
```

Comment:
Do we want first location to be used, as it is the "longest"? I guess multiple calls to setLocation have happened before some code was generated...